CHANGELOG.md

```diff
@@ -103,19 +103,26 @@
   ```
 * Added `RealmValueType` enum that contains all the possible types that can be wrapped by a `RealmValue`. (PR [#1469](https://github.com/realm/realm-dart/pull/1469))
 * Added support for accessing `Set` and `Map` types using the dynamic object API - `obj.dynamic.getSet/getMap`. (PR [#1533](https://github.com/realm/realm-dart/pull/1533))
-
+* Improve performance of object notifiers with complex schemas and very simple changes to process by as much as 20% ([PR #7424](https://github.com/realm/realm-core/pull/7424)).
```

Comment:
I think for those we'd probably want to just have `Core v14.2.0` or similar, since it's not super easy to determine whether the PR link is going to be to the dart repo or the core one.

```suggestion
* Improve performance of object notifiers with complex schemas and very simple changes to process by as much as 20% (Core v14.2.0).
```

Reply:
I'm not a big fan of dropping information like this. I know we did in the past, and I'll update to match in this PR, but I think we should reconsider. 

These are links, easy to follow, if you want more info, and it is actually a bit less editing to leave them in.